Imperatus.rte/Loadouts.ini

```diff
@@ -88,15 +88,13 @@ AddLoadout = Loadout
 		PresetName = Imperatus.rte/Chunker SP-44
 
 	AddCargoItem = AHuman
-		PresetName = Imperatus.rte/All Purpose Robot
+		PresetName = Imperatus.rte/Combat Robot
 	AddCargoItem = HDFirearm
 		PresetName = Imperatus.rte/Bullpup AR-14
 	AddCargoItem = HDFirearm
 		PresetName = Imperatus.rte/Chunker SP-44
 	AddCargoItem = TDExplosive
 		PresetName = Imperatus.rte/Shredder SB-08 Bandolier
-	AddCargoItem = HDFirearm
```

Comment:
Why remove the medkit from this loadout? I deliberately decided to make loadouts contain medkits to make them more viable for players.
Remember, loadouts are for players, not the AI, so all you're doing is making it a hassle for the player to manually add a medkit here.

Reply:
I figured Impies aren't the kind of faction to (have to) rely on medikits, especially in the case of a Combat Robot. All this time I've treated Loadouts as a thematical presentation of how factions would equip themselves. If it's such a big deal then I can add it back.

By the way, since when haven't the AI been using loadouts? Don't they spawn actors based on them anymore?

Another change I'm particularly unhappy with is the introduction of the Medic loadout, which somehow emphasizes the apparent universal need of medikits in the game. And not a single one of them is a Medic Drone?!

Reply:
I agree that thematically, medkits for robots feels… weird.

in terms of load out stuff- having a medic drone load out doesn’t make much sense if they’re for human players to use, as it’s a single item.

We changed AI to not use load outs because there was considered too much contradiction in what the AI would need vs players. For example, it was considered that breaching tools shouldn’t be in load outs bust a significant proportion of AI actors should have one. We can still “hand roll” load outs programmatically via Lua, and I think the plan for future will be to formalise that a bit.

Reply:
Yeah, unfortunately the formalization of unit spawning it's yet another thing that's beyond the scope for now... I think it would have been best to find a middle ground with loadouts for now, letting both player and AI use them, mainly for thematical purposes so that each loadout list could reflect the nature of the faction

Reply:
Even in pre4 conquest didn't use loadouts. Bunker breach used it half the time for no good reason really, and we got rid of it cause it meant too few breaching tools for attackers haha.

As for the medic loadout, the thing is that a large number of AI units now have medikits via activity spawns, which improves their survivability on a regular basis. Human players can obviously do this too, but the loadouts didn't support this, which gets in the way of their usefulness. I agree that thematically it's a bit weird for imperatus robots to have medikits, but that's really just a matter of them not being called repair kits imo.

Reply:
I think it's thematically wrong for Imperatus to waste time repairing themselves :V